book/graphics.md

```diff
@@ -700,7 +702,7 @@ were `about:blank`.
 2-7 *Alternate text direction*. Not all languages read and lay out from left
 to right. Arabic, Persian, and Hebrew are good examples of right-to-left
 languages. Implement basic support for this with a command-line flag to your
-browser.^[Once we get to [Chapter 4](html.md) you could write it in terms of
+browser.^[Once we get to [Chapter 4](html.md) you could instead use
```

Comment:
"Write it in terms of" makes it sound like your implementation should use a `dir` attribute, which is a little weird.